openassessment/xblock/submission_mixin.py

```diff
@@ -148,13 +168,22 @@ def submit(self, data, suffix=''):  # pylint: disable=unused-argument
                 logger.exception(msg)
                 status_tag = 'EUNKNOWN'
                 status_text = self._(u'API returned unclassified exception.')
-            else:
-                status = True
-                status_tag = submission.get('student_item')
-                status_text = submission.get('attempt_number')
```

Comment:
Why did you delete this?

Reply:
That's now moved to `_create_submisison_response()`, which is called in both paths of the `try:` block earlier in the funciton.

The only remaining codepath is exception handling which explicitly overrides those tags so I thought it was redundant.